portal/account/index_reset.php

```diff
@@ -0,0 +1,170 @@
+<?php
+/**
+ * Credential Changes
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2019 Jerry Padgett <sjpadgett@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
```

Comment:
Set up PR here which will support csrf in the portal:
https://github.com/openemr/openemr/pull/2718

Will take 3 lines of code to then support in this script. I'll show them below.